go/common/types.go

```diff
@@ -197,3 +199,15 @@ type LogSubscription struct {
 	// A subscriber-defined filter to apply to the stream of logs.
 	Filter *filters.FilterCriteria
 }
+
+// IDAndEncLog pairs an encrypted log with the ID of the subscription that generated it.
+type IDAndEncLog struct {
+	ID     rpc.ID
```

Comment:
Maybe call this field subID or something, casual reader would assume that's an ID for the log message